### PR TITLE
added solution

### PR DESCRIPTION
**Pull request by tanchak123**

### CODE REVIEW

File: src/main/java/core/basesyntax/BinaryString.java [hunk 1]
```diff
@@ -2,11 +2,19 @@
...
-    /**
-     * Напишите метод toBinaryString(), который принимает на вход целое число value,
-     * а возвращает String с представлением этого числа в двоичном виде.
-     */
     public String toBinaryString(int value) {
-        return null;
+        if (value == 0) {
+            return "0";
+        }
+        String result = "";
```
Comment (Tarabarnik) [inline src/main/java/core/basesyntax/BinaryString.java]: can you create the solution without `result` variable?

**Review by Tarabarnik — APPROVED** [review]

File: src/main/java/core/basesyntax/BinaryString.java (hunk 1 above)
Comment (tanchak123) [reply src/main/java/core/basesyntax/BinaryString.java]: done

File: src/main/java/core/basesyntax/BinaryString.java [hunk 2]
```diff
@@ -2,11 +2,16 @@
...
-    /**
-     * Напишите метод toBinaryString(), который принимает на вход целое число value,
-     * а возвращает String с представлением этого числа в двоичном виде.
-     */
     public String toBinaryString(int value) {
-        return null;
+        if (value == 0) {
+            return "0";
+        }
+        StringBuilder itog = new StringBuilder();
```
Comment (Tarabarnik) [inline src/main/java/core/basesyntax/BinaryString.java]:
```suggestion
        StringBuilder result = new StringBuilder();
```
let's write in English :)

File: src/main/java/core/basesyntax/BinaryString.java [hunk 3]
```diff
@@ -2,11 +2,16 @@
...
+        while (value > 0) {
+            itog = itog.append(value % 2);
```
Comment (Tarabarnik) [inline src/main/java/core/basesyntax/BinaryString.java]:
```suggestion
            itog.append(value % 2);
```

File: src/main/java/core/basesyntax/BinaryString.java (hunk 2 above)
Comment (tanchak123) [reply src/main/java/core/basesyntax/BinaryString.java]: )))))

**Review by Tarabarnik — APPROVED** [review]

**Pull request closed without merging**